CaImAn/caiman_utilities.py

```diff
@@ -152,4 +155,37 @@ def set_advanced_parameters(
     #keep only used keys in andvanced parameters
     advanced_parameters = {key: advanced_parameters[key] for key in advan_param_keys_used}
 
-    return [param, advanced_parameters]
\ No newline at end of file
+    return [param, advanced_parameters]
+
+
+def create_param_attribut_to_save(
```

Comment:
This function seems to be the same for minian and caiman, so I would put it in utilities.py

Reply:
Yes they do really similar stuff. But for caiman and minian they don't do the same thing so that is why I did two different one.

I didn't really think of merging them together but I can if you want

Reply:
Oh, it's because of the advanced parameters

Reply:
Yes. The advanced parameters are not the same in both.

Also as a side not I want to clean up the code for minian and caiman and try harmonized the way I do things in both of them. Because with caiman and minian not being the same and adding the fact that I was tickering with the code, therefore doing stuff differently, it is really confusing sometime to work with both of them......

and I want to add a comment thing (a class? or a dictionary?) with Names that come from danse like attributes Keys. (something similar as nsCrop::Parameters::CROP_COORDINATES exemple)